.mergify.yml

```diff
@@ -0,0 +1,31 @@
+pull_request_rules:
+  - name: ask to resolve conflict
+    conditions:
+      - conflict
+    actions:
+        comment:
+          message: |
+            This pull request is now in conflicts. Could you fix it @{{author}}? 🙏
```

Comment:
`author` won't be resolved correctly -> https://github.com/Mergifyio/mergify-engine/issues/1096

Reply:
See https://github.com/v1v/apm-pipeline-library/pull/19#issuecomment-802306489

![image](https://user-images.githubusercontent.com/2871786/111699059-87e98280-882f-11eb-9059-2a1cf3bef9ee.png)